react/client/src/components/Button.tsx

```diff
@@ -1,61 +1,54 @@
-import styled from 'styled-components';
+import React from 'react';
```

Comment:
@aahvocado This is probably closer to what you were looking for, generic React components right? Let me know and I'll finish converting and replacing the rest

Reply:
Yep, this is what I had in mind -- but if you also found something more clear and clever, I'm all for it

Reply:
I did start with a different approach using custom material-ui classes, but this is way more intuitive. Will continue converting the rest to generic components. Thanks!